VENDORING.md

```diff
@@ -0,0 +1,238 @@
+# Perform the vendoring of the Kata Containers project
+
+* [Vendoring tool](#vendoring-tool)
+    * [How to install dep](#how-to-install-dep)
+    * [Overview of the main commands](#overview-of-the-main-commands)
+* [Vendoring use cases](#vendoring-use-cases)
+    * [Initialize vendoring of the repository](#initialize-vendoring-of-the-repository)
+    * [Vendor a newly imported package](#vendor-a-newly-imported-package)
+    * [Update a vendored package](#update-a-vendored-package)
+    * [Remove a vendored package](#remove-a-vendored-package)
+    * [Test a pending pull request](#test-a-pending-pull-request)
+
+## Vendoring tool
+
+[`dep`](https://github.com/golang/dep) is the vendoring tool chosen for this
+project. The choice has been made to use this tool because it is fairly simple
+and very powerful for some corner cases. Furthermore, this tool is expected to
+become the official vendoring tool for Golang.
+
+### How to install dep
+
+Installing this tool is simple, use the following command:
+```bash
+go get -u github.com/golang/dep/cmd/dep
+```
+
+Make sure your `PATH` has the path `$GOPATH/bin`, including all Go binaries,
+and particularly `dep`, otherwise add it with the following command:
+```bash
+export PATH=$GOPATH/bin:$PATH
+```
+
+### Overview of the main commands
+
+`dep init`: Initialize a new project with manifest and lock files.
+`dep ensure`: Ensure a dependency is safely vendored in the project.
+`dep prune`: Prune the vendor tree of unused packages.
+
+## Vendoring use cases
+
+Here will be listed the different vendoring use cases a developer could
+encounter when developing for Kata Containers.
+
+### Initialize vendoring of the repository
+
+If no prior vendoring has been performed on the repository, you need to
+initialize it with the following command:
+```bash
+dep init
```

Comment:
Can you add a leading `$ ` prompt here and below to show this is a non-`root` command to run?

Reply:
done